.github/workflows/ci.yaml

```diff
@@ -211,19 +216,42 @@ jobs:
         #   CARGO_TARGET_ARMV7_UNKNOWN_LINUX_GNUEABIHF_LINKER: ${{ matrix.cc }}
         #   CARGO_TARGET_X86_64_UNKNOWN_LINUX_MUSL_LINKER: ${{ matrix.cc }}
 
-      - name: Compress artifact (Not Windows)
-        if: matrix.os != 'windows-latest'
-        run: zip --junk-paths ${{ matrix.platform }} ./target/${{ matrix.target }}/release/clarinet
+      - name: Build (Windows)
+        if: matrix.os == 'windows-latest'
+        run: |
+          cargo wix -v
+          ls target/wix
 
-      - name: Compress artifact (Windows)
+      - name: Code Sign (Windows)
         if: matrix.os == 'windows-latest'
-        run: Compress-Archive -Path ./target/${{ matrix.target }}/release/clarinet.exe -DestinationPath ${{ matrix.platform }}
+        run: |
+          gci env:* | sort-object name
+          cargo wix sign --verbose --nocapture --bin-path "C:/Program Files (x86)/Windows Kits/10/bin/10.0.19041.0/x86/"
```

Comment:
Getting some PTSD looking at this. You may way to check out this script https://github.com/blockstack/blockstack-browser/blob/bd664e21e550ade00615670a9fa97780ac5ad780/native/windows/signtool.cmd
```cmd
set signtool_search="%ProgramFiles(x86)%\Windows Kits\10\bin\10*\x86\signtool.exe"

for /f "usebackq delims=" %%i in (`powershell -command "& { @((Resolve-Path -Path \"%signtool_search%\").Path)[0] }"`) do (
  echo Using signtool: %%i
  set signtool_path=%%i
  goto found_signtool
)
echo signtool.exe not found
exit /b 2

:found_signtool
"%signtool_path%" %*
exit /b !errorlevel!
```

Usage here: https://github.com/blockstack/blockstack-browser/blob/bd664e21e550ade00615670a9fa97780ac5ad780/package.json#L187
`signtool.cmd sign /t http://timestamp.verisign.com/scripts/timstamp.dll /n Hiro /f signcertfile.pfx file.msi`

Reply:
First, _I'm sorry_ you had to go through this. Second, thank you for the tip!